### PR TITLE
Gjl774/example scripts

### PR DESCRIPTION
**Pull request by gabelepoudre**
# Pull Request

## Describe your changes

Added sad fish

also added example script

## Suggested reviewers (also add these to "reviewers" on the right menu)

Greta

## Quality Assurance checklist (for code PRs)
- [x] I have manually tested my code and fixed all bugs I am aware of
- [x] I have manually reviewed my code
- [x] I have adhered to the [Style Guide](style.md) within reason
- [x] I have adhered to the [Practices Guide](practices.md) and commented all code

## Additional Notes

...

### CODE REVIEW

File: mystery-deckbuilder/Assets/Scripts/Examples/Test.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Test : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        // do later
+    }
```
Comment (gabelepoudre) [inline mystery-deckbuilder/Assets/Scripts/Examples/Test.cs:11]: NIT: This isn't really a good comment

File: mystery-deckbuilder/Assets/Scripts.meta [hunk 2]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: 8c850bb98f6982147b12d75e390944c4
+folderAsset: yes
+DefaultImporter:
+  externalObjects: {}
+  userData: 
+  assetBundleName:
```
Comment (gabelepoudre) [inline mystery-deckbuilder/Assets/Scripts.meta:7]: Do we need these meta scripts

**Review by gabelepoudre — COMMENTED** [review]
Overall, looks okay

File: mystery-deckbuilder/Assets/Scripts/Examples/Test.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+    // Update is called once per frame
+    void Update()
+    {
+
```
Comment (g-stru) [inline mystery-deckbuilder/Assets/Scripts/Examples/Test.cs:16]: nothing is here ??????

**Review by g-stru — APPROVED** [review]
Overall just okay

File: mystery-deckbuilder/Assets/Scripts/Examples/Test.cs (hunk 3 above)
Comment (gabelepoudre) [reply mystery-deckbuilder/Assets/Scripts/Examples/Test.cs:16]: That's okay

**Pull request merged**